shared/src/utils/validators/apply-decorators.ts

```diff
@@ -0,0 +1,32 @@
+// Source: https://github.com/nestjs/nest/blob/a448f53b7746d35bf25a18f1759c971e5b7fea1c/packages/common/decorators/core/apply-decorators.ts
+/**
+ * Function that returns a new decorator that applies all decorators provided by param
+ *
+ * Useful to build new decorators (or a decorator factory) encapsulating multiple decorators related with the same feature
+ *
+ * @param decorators one or more decorators (e.g., `ApplyGuard(...)`)
+ *
+ * @publicApi
+ */
+export function applyDecorators(
```

Comment:
```suggestion
/**
 * @param decorators one or more decorators (e.g., `ApplyGuard(...)`)
 * @returns a new decorator that applies all the provided decorators
 * 
 * Useful to build new decorators (or a decorator factory) encapsulating multiple decorators
 */
export function combineDecorators(
```

or `mergeDecorators()`

and renaming the file appropriately.